Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -148,6 +154,8 @@ class FirefoxHomeViewController: UICollectionViewController, HomePanel {
     }()
 
     var pocketStories: [PocketStory] = []
+    var isRecentlySavedSectionHidden = false
+    var isRecentlySavedSectionEnabled = AppConstants.IS_RECENTLY_SAVED_SECTION_ENABLED
```

Comment:
I would argue that accessing `AppConstants.IS_RECENTLY_SAVED_SECTION_ENABLED` in the code directly is probably the way to go here. This var just becomes a random middleman that we'd need to maintain.